base/applications/dxdiag/lang/zh-TW.rc

```diff
@@ -26,7 +27,7 @@ STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_CLIPCHILDREN
 FONT 9, "新細明體"
 BEGIN
     LTEXT "此工具會報告安裝在您系統的 ReactX 元件和驅動程式的詳細資料。", IDC_STATIC, 5, 0, 443, 17
-    LTEXT "如果你知道那個區域造成問題，單擊上面相應的標籤。否則，你可以使用「下一頁」按鈕來順序訪問每一頁。", IDC_STATIC, 5, 15, 443, 25
+    LTEXT "如果您知道造成問題的區域，請點擊上面相應的標籤。否則，您可以使用 [下一頁] 按鈕來順序訪問每一頁。", IDC_STATIC, 5, 15, 443, 25
```

Comment:
why use [] instead of 「」🤣

Reply:
1. My practice
2. This is also how windows do

Reply:
If you guys can actually use a translation that can be improved compared to Windows, this would be great!

Reply:
「」 is Chinese quotes, which is the newest standard but actually not popularized (at least for now) 🤣

Reply:
actually I usually use 「」 on Chinese quotes on other translation project, just I don't use it here. 😅

btw, what if I change most of quotation mark used on quote like this into 「」? I think this can let translation more localised. (if ok I do next PR just for this)

Reply:
I think it would be great to use the latest standard. I'm not a professional in translation. It's up to you🤣

Reply:
ok, I keep this change first. I may change it back when I decided to change quotation mark to 「」 in later PR.